src/rollup.ts

```diff
@@ -36,21 +37,38 @@ function rewriteInputsNamespace(code: string) {
 export async function bundleStyles({
   minify = false,
   path,
-  theme
+  theme,
+  aliases
 }: {
   minify?: boolean;
   path?: string;
   theme?: string[];
-}): Promise<string> {
+  aliases?: Map<string, string>;
+}): Promise<{contents: string; files: Set<string>}> {
+  const files = new Set<string>();
+  const assets = {
+    name: "resolve CSS assets",
+    setup(build: PluginBuild) {
+      build.onResolve({filter: /^\w+:\/\//}, (args) => ({path: args.path, external: true}));
+      build.onResolve({filter: /./}, (args) => {
+        if (args.path.endsWith(".css") || args.path.match(/^[#.]/)) return;
+        files.add(args.path);
+        const path = join("..", aliases?.get(args.path) ?? join("_file", args.path));
```

Comment:
Another big advantage of the `resolve` hook is that `bundleStyles` won’t have to know about the `_file` directory — that logic can be supplied by the build command.

And for the preview command, ideally we’d supply a `resolve` hook that does `?sha=…` so that module replacement works when you update a file referenced by a local stylesheet?